crates/astria-core/src/primitive/v1/mod.rs

```diff
@@ -923,6 +921,18 @@ mod tests {
         let _ = address.into_raw();
     }
 
+    #[test]
+    fn pbjson_base64_should_be_url_safe() {
```

Comment:
Is this test necessary at this level (given the premise that downstream users should be able to rely on the correct implementation of upstream libraries)?

Reply:
I think tests like these are worthwhile anywhere we don't just do a standard `derive(Serialize)`, since it'd be easy to regress otherwise.

This is even more the case now we're going to be using a less obvious way to serialize the domain `RollupId` type.

Potentially other snapshot tests would catch this, but I don't like to rely on those to test this functionality, since that too may change over time.